pkg/server/plugin/datastore/datastore.go

```diff
@@ -130,22 +139,20 @@ type ListBundlesResponse struct {
 }
 
 type ListNodeSelectorsRequest struct {
-	// When enabled, read-only connection will be used to connect to database read instances. Some staleness of data will be observed.
-	TolerateStale bool
-	ValidAt       *time.Time
+	DatabasePreference
```

Comment:
Is this being embedded just save a little stuttering on the field name and type. Is there another reason that I'm missing? If not, embedding doesn't seem like a good idea. For example, if `DatabasePreference` gained a String() method, which seems practical, this request object would suddenly have a non-sensical String() implementation.

Reply:
Fair. I was trying to reduce stuttering; however, prolly not worth the future cost here